src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -827,9 +764,8 @@ def _set_fs_copy_params(self,
             self.fs_copy_cmd.set_fs_copy_params(
                 src=str(src_path))
         elif src_type == "DAOS_UUID":
-            param = self._format_daos_path(src_pool, src_cont, src_path)
             self.fs_copy_cmd.set_fs_copy_params(
-                src=param)
+                src=self.format_daos_path(src_pool, src_cont, src_path))
```

Comment:
(pylint-no-member) Instance of 'DataMoverTestBase' has no 'format_daos_path' member